Source/WebInspectorUI/UserInterface/Controllers/AnimationManager.js

```diff
@@ -126,7 +126,7 @@ WI.AnimationManager = class AnimationManager
         if (!animation)
             return;
 
-        // COMPATIBILITY (iOS 18.X, macOS 15.X): `Animation.effectChanged` removed the `effect` parameter in favor of `Animation.requestEffect`.
```

Comment:
there's also a lot more changes for `COMPATIBILITY (iOS X.Y, macOS X.Y)` that need to be updated